xml/System.Runtime.Intrinsics.Arm/Sve.xml

```diff
@@ -34115,7 +34115,7 @@
           <para>void svprfb(svbool_t pg, const void *base, enum svprfop op)</para>
           <para>  PRFB op, Pg, [Xbase, #0, MUL VL]</para>
         </summary>
-        <remarks>To be added.</remarks>
+        <remarks>This method was renamed to in <see cref="M:System.Runtime.Intrinsics.Arm.Sve.Prefetch8Bit(System.Numerics.Vector{System.UInt8},System.Void*,System.Runtime.Intrinsics.Arm.SvePrefetchType)" /> .NET 10.</remarks>
```

Comment:
Do we need these remarks? These were experimental APIs where the previous names were incorrect and so we're just fixing them to be the correct name